R/DBConnection.R

```diff
@@ -106,6 +106,39 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' @export
+dbBreak <- structure(list(), class = "dbi_break_sentinel")
+
+dbFetchChunked <- function(rs, n, callback) {
```

Comment:
@jcheng5 I get the name change, but I'm less sure on the need for `dbFetchChunkedQuery` to be exported. Unlike `dbFetch`, this is *not* something that individual backends should implement themselves. This is simply a pattern of fetching that uses the `dbFetch`, which is of course implemented by each backend. So no need for export there (it might actually confuse backend authors in my opinion). This is also not something that the end user should be calling directly -- that is why `dbGetChunkedQuery` exists (which is essentially just a wrapper around `dbFetchChunkedQuery`, which guarantees the closing of the result set). So no need for exporting here either. Am I missing something?